build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -628,6 +641,25 @@ message Directory {
   repeated SymlinkNode symlinks = 3;
 }
 
+// A single property for [FileNodes][build.bazel.remote.execution.v2.FileNode],
+// [DirectoryNodes][build.bazel.remote.execution.v2.DirectoryNode], and
+// [SymlinkNodes][build.bazel.remote.execution.v2.SymlinkNode]. The server is
+// responsible for specifying the property `name`s that it accepts. If
+// permitted by the server, the same `name` may occur multiple times.
+//
+// The server is also responsible for specifying the interpretation of
```

Comment:
I think this paragraph is no longer true. All the properties are rigorously defined in the md file.

Reply:
yes, i think so